tests/manifold/projection_manifold_01.cc

```diff
@@ -0,0 +1,74 @@
+//-------------------------------------------------------------------
+//    Copyright (C) 2016 by the deal.II authors.
```

Comment:
still it looks different from what i quoted. I am not specialist in different flavours of licenses, though.